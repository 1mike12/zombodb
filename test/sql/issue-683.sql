SELECT * FROM products WHERE products ==> 'Box';
SELECT * FROM products WHERE products ==> 'Bo?';
SELECT * FROM products WHERE products ==> 'Bo*';