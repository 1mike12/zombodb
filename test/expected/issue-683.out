SELECT * FROM products WHERE products ==> 'Box';
 id | name |               keywords               |         short_summary          |                                  long_description                                   | price | inventory_count | discontinued | availability_date 
----+------+--------------------------------------+--------------------------------+-------------------------------------------------------------------------------------+-------+-----------------+--------------+-------------------
  4 | Box  | {wooden,box,"negative space",square} | Just an empty box made of wood | A wooden container that will eventually rot away.  Put stuff it in (but not a cat). | 17000 |               0 | t            | 07-01-2015
(1 row)

SELECT * FROM products WHERE products ==> 'Bo?';
 id | name |               keywords               |         short_summary          |                                  long_description                                   | price | inventory_count | discontinued | availability_date 
----+------+--------------------------------------+--------------------------------+-------------------------------------------------------------------------------------+-------+-----------------+--------------+-------------------
  4 | Box  | {wooden,box,"negative space",square} | Just an empty box made of wood | A wooden container that will eventually rot away.  Put stuff it in (but not a cat). | 17000 |               0 | t            | 07-01-2015
(1 row)

SELECT * FROM products WHERE products ==> 'Bo*';
 id | name |               keywords               |         short_summary          |                                  long_description                                   | price | inventory_count | discontinued | availability_date 
----+------+--------------------------------------+--------------------------------+-------------------------------------------------------------------------------------+-------+-----------------+--------------+-------------------
  4 | Box  | {wooden,box,"negative space",square} | Just an empty box made of wood | A wooden container that will eventually rot away.  Put stuff it in (but not a cat). | 17000 |               0 | t            | 07-01-2015
(1 row)

